mopidy/mpd/translator.py

```diff
@@ -86,6 +87,11 @@ def track_to_mpd_format(track, position=None, stream_title=None):
     if track.disc_no:
         result.append(('Disc', track.disc_no))
 
+    if track.last_modified is not None:
+        datestring = datetime.datetime.fromtimestamp(
```

Comment:
Use `datetime.datetime.utcfromtimestamp` instead, and this will no longer depend on the local time zone.

Then you can safely just append "Z" to the returned string yourself, since the returned timestamp will always be in the UTC time zone.